snapshot.css

```diff
@@ -5621,10 +5964,10 @@ html:lang(ar)
   border-width: 2px;
   border-style: solid;
   border-color: #0168b3;
-  top: -5px;
-  left: -5px;
-  right: -5px;
-  bottom: -5px;
+  top: calc(-1 * $focus-ring-offset);
```

Comment:
This should be fixed. The path to the source is in the class name:  `.draft-packages-likert-scale-legacy-KaizenDraft-LikertScaleLegacy-styles-module__container--1Iqv-.focus-visible`

The string literal `$focus-ring-offset` is being used within a calc.